.devcontainer/Dockerfile

```diff
@@ -3,7 +3,7 @@
 # Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
```

Comment:
@bergmeister, your last commit had 1 failures in `PowerShell-CI-windows`
ConsoleHost unit tests.ApartmentState WPF tests.WPF requires STA and will work
```powershell
The following exception occurred while retrieving member "Load": "Could not load type 'MS.Internal.SecurityCriticalDataForSet`1' from assembly 'WindowsBase, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'."
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\ConsoleHost.Tests.ps1: line 733
733:             $Window = [System.Windows.Markup.XamlReader]::Load($reader)

```

Reply:
I can reproduce this locally, the constructor of `System.Windows.Markup.XamlReader` throws, which is probably a new WPF bug, will dig further
```pwsh
add-type -AssemblyName presentationframework
$xaml = [xml]@"
            <Window
                xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
                xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
                x:Name="Window" Title="Initial Window" WindowStartupLocation = "CenterScreen"
                Width = "400" Height = "300" ShowInTaskbar = "True">
            </Window>
"@
$reader = [System.Xml.XmlNodeReader]::new($xaml)
[System.Windows.Markup.XamlReader]::Load($reader)
```